### PR TITLE
312 robot bodybrain ea get genotype from database

### LINKED ISSUE #247: Add an example for recovery
Show how to perform recovery when you are using a database.

### LINKED ISSUE #312: Robot bodybrain EA get genotype from database
Currently it's a copy of without database, but that is incorrect.

### PR DESCRIPTION
**Pull request by surgura**
* Add rerun to bodybrain and cmaes examples
* Fix deserialization of EA parameters type.

closes #312
closes #247

### CODE REVIEW

File: examples/evaluate_multiple_interacting_robots/README.md
Comment (oliverweissl) [inline examples/evaluate_multiple_interacting_robots/README.md:1]: what changed here?

File: examples/custom_terrain/README.md
Comment (oliverweissl) [inline examples/custom_terrain/README.md:1]: this is probably due to the ide

File: examples/experiment_setup/README.md
Comment (oliverweissl) [inline examples/experiment_setup/README.md:1]: again lol

**Review by oliverweissl — COMMENTED** [review]
some files changed but i dont see why.
is this a specific formatting choice?

File: examples/evaluate_multiple_interacting_robots/README.md
Comment (surgura) [reply examples/evaluate_multiple_interacting_robots/README.md:1]: it's just me adding a newline at the of the file. github displays it weirdly

**Review by oliverweissl — APPROVED** [review]

**Pull request merged**